app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/conversation/ConversationSheetContent.kt

```diff
@@ -15,16 +15,20 @@ import com.wire.kalium.logic.data.user.UserId
 @Composable
 fun ConversationSheetContent(
     conversationSheetState: ConversationSheetState,
-    onMutingConversationStatusChange: (MutedConversationStatus) -> Unit,
+    onMutingConversationStatusChange: () -> Unit,
     addConversationToFavourites: () -> Unit,
     moveConversationToFolder: () -> Unit,
     moveConversationToArchive: () -> Unit,
     clearConversationContent: () -> Unit,
     blockUser: (BlockUserDialogState) -> Unit,
     unblockUser: (UnblockUserDialogState) -> Unit,
     leaveGroup: (GroupDialogState) -> Unit,
-    deleteGroup: (GroupDialogState) -> Unit
+    deleteGroup: (GroupDialogState) -> Unit,
+    isBottomSheetVisible: () -> Boolean = { true }
```

Comment:
Quick question, why we need the default value of this `isBottomSheetVisible`  to be visible by default ?

Reply:
that is actually a good question, fixed it :)